pyaedt/edb.py

```diff
@@ -60,8 +61,8 @@ class Edb(object):
     isaedtowned : bool, optional
         Whether to launch ``edb_core`` from HFSS 3D Layout. The
         default is ``False``.
-    oproject : optional
-    
+    oproject : str, optional
+        Name of the project to open.
```

Comment:
```suggestion
    oproject : optional
        Reference to the AEDT project object.
```
This parameter is an object from the AEDT application. It does not really have any type associated with it. Feel free to the reword my suggested description.